shared/src/main/scala/mlscript/ConstraintSolver.scala

```diff
@@ -639,8 +643,12 @@ class ConstraintSolver extends NormalForms { self: Typer =>
         (lhs.lb, rhs.lb) match {
           case (Some(l), Some(r)) => rec(l, r, false)
           case (Some(l), None) =>
-            if (lhs.prov.loco.isEmpty || rhs.prov.loco.isEmpty) reportError()
-            else reportError(S(msg"is not mutable"))(
+            println(s"RHS not mutable! in $lhs <- $rhs")
```

Comment:
Removing the extra exclamation mark?

```suggestion
            println(s"RHS not mutable in $lhs <- $rhs")
```

Reply:
nope